catroid/src/org/catrobat/catroid/ui/dialogs/NewDataDialog.java

```diff
@@ -57,6 +57,7 @@
 	public static final String DIALOG_FRAGMENT_TAG = "dialog_new_data_catroid";
 	Spinner spinnerToUpdate;
```

Comment:
class member without a visibility modifier get a default visibility - which means that they can be accessed by any class in the same package. In this example all your member variables (spinnerToUpdate, dialogType and userVariableIfCancell [why cancell instead of cancel?]) are exposed to the package. Please make them private!